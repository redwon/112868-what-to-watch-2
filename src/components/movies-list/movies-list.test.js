import React from 'react';
import renderer from 'react-test-renderer';

import MoviesList from './movies-list';

const mockMovies = [
  {
    name: `name`,
    image: `image`
  },
  {
    name: `name`,
    image: `image`
  }
];

it(`renders correctly`, () => {
  const tree = renderer
    .create(<MoviesList
      movies={mockMovies}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
