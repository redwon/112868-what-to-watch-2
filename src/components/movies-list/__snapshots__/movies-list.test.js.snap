// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="name"
        height="175"
        src="image"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        name
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="name"
        height="175"
        src="image"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        name
      </a>
    </h3>
  </article>
</div>
`;
